static/js/click_handlers.js

```diff
@@ -461,8 +461,8 @@ exports.initialize = function () {
     });
 
     // this will hide the alerts that you click "x" on.
-    $("body").on("click", ".alert .exit", function () {
-        var $alert = $(this).closest(".alert");
+    $("body").on("click", ".alert-box>div .exit", function () {
```

Comment:
We'd usually put spacing around the `>` here, but I'm also curious why this change is needed --- is there a new type of `.exit` widget?

Reply:
The stacktrace popup doesn't use the `.alert` styling, so we need this for the exit button to work. (This is also why I extracted some bits out of `.alert` to `.alert-base` to share with stacktrace)